test/src/main/java/org/openjdk/skara/test/TestIssue.java

```diff
@@ -182,19 +182,22 @@ public boolean isFixed() {
 
     @Override
     public void addLabel(String label) {
+        labels = null;
         var now = ZonedDateTime.now();
         store.labels().put(label, now);
         store.setLastUpdate(now);
     }
 
     @Override
     public void removeLabel(String label) {
+        labels = null;
         store.labels().remove(label);
         store.setLastUpdate(ZonedDateTime.now());
     }
 
     @Override
     public void setLabels(List<String> labels) {
+        this.labels = null;
```

Comment:
This method should update the `labels` field instead of setting it to null.

Reply:
The same with JiraIssue